source/renderer/app/types/sidebarTypes.js

```diff
@@ -6,4 +6,5 @@ export type SidebarWalletType = {
   isConnected: boolean,
   isRestoreActive?: boolean,
   restoreProgress?: number,
+  isLegacy?: ?boolean,
```

Comment:
@yakovkaravelov please use the following type declaration:
```
isLegacy: boolean,
```

Reply:
Fixed.